### DISCUSSION TITLE
How to do 'getTrProps' in react-table v7

### DISCUSSION THREAD

**Discussion by mikefedorko** [TanStack/table#2440] [General]
I have to check if field isCancelled equal true, then make row red color.
Can somebody explain me how to make something like that in v7 react-table: 
`getTrProps={(_, { original }) => original.isCancelled ? { style: { background: 'red' } } : {}}`
I couldn't find how to do this in documentation.

**Comment by arditnurcaj** [comment]
Hi @mikefedorko check this one https://codesandbox.io/s/cool-frog-u5y8z?file=/src/App.js.

P.S the solution you are looking for is written from lines 65-81, 

Good luck,
Ardit.
